src/serverlessFunction/functionImpl.ts

```diff
@@ -126,19 +127,19 @@ export class ServerlessFunctionImpl implements ServerlessFunction {
         return functionList;
     }
 
-    getFunctionStatus(funcData: FunctionContent, deployedFunctions: FunctionObject[]): FunctionStatus {
+    getFunctionStatus(funcData: FunctionContent, deployedFunctions: FunctionObject[]): [FunctionStatus, string] {
```

Comment:
Add the URL as an optional field of `FunctionStatus` instead of a separate return item